src/allmydata/interfaces.py

```diff
@@ -351,6 +351,10 @@ def slot_testv_and_readv_and_writev(
     ):
         """
         :see: ``RIStorageServer.slot_testv_readv_and_writev``
+
+        While the interface mostly matches, test vectors are simplified.
+        Instead of a tuple ``(start, offset, operator, data)`` they are just
+        ``(start, data_length, data)`` with operator implicitly being ``b"eq"``.
```

Comment:
Maybe `read_length` is a better name than `data_length` because of the `length=1` / `data=b""` case.  The length tells the server how much data to read for comparison, not the length of the data supplied.

Reply:
Indeed.